owncloudApp/src/main/res/values/strings.xml

```diff
@@ -763,6 +763,8 @@
     <string name="release_notes_4_3_0_subtitle_accessibility_improvements">Some improvements to make the application more accessible</string>
     <string name="release_notes_4_3_0_title_show_app_provider_icon_from_endpoint">New icons in "Open in (web)" option on the operations menu</string>
     <string name="release_notes_4_3_0_subtitle_show_app_provider_icon_from_endpoint">More appropriate icons have been added to the "Open in (web)" option on the operations menu</string>
+    <string name="release_notes_4_3_0_title_report_feedback_central_github">Changes in the Feedback section</string>
+    <string name="release_notes_4_3_0_subtitle_report_feedback_central_github">A new dialog has been added when Feedback section is opened in the drawer menu and Settings. Two options for feedback will be offered within the dialog: Central Community and Github. Clicking on the links will redirect you to these pages.</string>
```

Comment:
i'd compress a little bit the explanation. Suggestion: `New dialog has been added to handle feedback: GitHub and the open forum Central will be our meeting point` . Open to discussion.

Reply:
Updated